entrypoint.sh

```diff
@@ -2,4 +2,6 @@
 cd $WORKDIR
 python manage.py collectstatic --noinput
 
+python manage.py search_index --rebuild -f
```

Comment:
Isso poderia virar uma rotina no celery? acredito que periodicamente faria mais sentido afinal você precisa ficar alimentando o elasticsearch com as novas informações inseridas no nlp logs

Reply:
Nesse caso não, esse comando é tipo o migrate para o ES

Reply:
esse comando precisa ser inicializado toda vez que o projeto for iniciado pelo entrypoint? é algo demorado ?